os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -746,8 +726,16 @@ static inline void amebasmart_spi_master_set_delays(FAR struct amebasmart_spidev
 static int amebasmart_spi_lock(FAR struct spi_dev_s *dev, bool lock)
 {
 	FAR struct amebasmart_spidev_s *priv = (FAR struct amebasmart_spidev_s *)dev;
-
+	/* When a lock is invoked, means that we are going to do some SPI operation,
+	   thus we should decide to suspend/resume SPI domain here
+	*/
+#ifdef CONFIG_PM
+	int domain = pm_domain_register("SPI");
+#endif
```

Comment:
Could you let me know why this is necessary? There is no `domain` variable usage.

Reply:
Sorry it was mistake, I will remove it.